tests/e2e/utils/kubeflow_uninstallation.py

```diff
@@ -149,10 +144,12 @@ def delete_component(
 def uninstall_remote_component(component_name, namespace):
     if check_helm_chart_exists(component_name, namespace):
         uninstall_helm(component_name, namespace)
-    if component_name == "aws-load-balancer-controller":
-        kubectl_delete(
+        if component_name == "aws-load-balancer-controller":
```

Comment:
we should instead pull the chart and delete the crds from there.
Cert manager is missing

Appreciate the fixes but can you please create a separate PR for these changes?